e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -514,84 +514,6 @@ describeIfIAMV2p1('projects API', () => {
         expect(response.body).to.have.length(2);
       });
     });
-
-    // must correspond to enum type in automate-gateway/.../common/rules.proto
-    const attributeValuesMap = new Map([
-      ['CHEF_SERVER', 'chef-server-dev.test'],
-      ['ENVIRONMENT', '_default'],
-      ['CHEF_ROLE', 'web'],
-      ['CHEF_TAG', 'test_tag'],
-      ['CHEF_POLICY_GROUP', 'test_group'],
-      ['CHEF_POLICY_NAME', 'test_name']
-      // leave out CHEF_ORGANIZATION since it's tested above
-    ]);
-
-    for (const [attribute, value] of attributeValuesMap) {
-      const targetNodeId = 'f6a5c33f-bef5-433b-815e-a8f6e69e6b1b';
-
-      it(`ingest rule with attribute ${attribute} gets applied to node`, () => {
-        const rule = {
-          id: 'rule-2',
-          name: 'rule 2',
-          type: 'NODE',
-          project_id: avengersProject.id,
-          conditions: [
-            {
-              attribute: attribute,
-              operator: 'EQUALS',
-              values: [value]
-            }
-          ]
-        };
-        // create rule with attribute value
-        cy.request({
-          headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
-          method: 'POST',
-          url: `/apis/iam/v2beta/projects/${rule.project_id}/rules`,
-          body: rule
-        });
-
-        // apply rules
-        cy.request({
-          headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
-          method: 'POST',
-          url: '/apis/iam/v2beta/apply-rules'
-        });
-        waitUntilApplyRulesNotRunning(100);
```

Comment:
This removed 12 apply rules. Which was replaced by one apply rules.

Reply:
I think it makes sense to move the whole `applying projects` describe block (starting L63) into that new test file. otherwise it feels a little random that we're testing rule update for one random attribute in this file, and the rest of the attributes in another file.

Reply:
I was thinking that the tests in the `applying projects` are testing the authz parts of the project update process. It is only using the client runs and organization as an example. I see tests for checking if the rules are staged and then applied, updating rules, and deleting rules.